Apps/Playground/App.tsx

```diff
@@ -6,7 +6,7 @@
  */
 
 import React, { useState, FunctionComponent, useEffect, useCallback } from 'react';
-import { SafeAreaView, StatusBar, Button, View, Text, ViewProps, Image } from 'react-native';
+import { SafeAreaView, StatusBar, Button, View, Text, ViewProps, Image, Platform } from 'react-native';
```

Comment:
I don't think this file is using `Platform` any more.
```suggestion
import { SafeAreaView, StatusBar, Button, View, Text, ViewProps, Image } from 'react-native';
```